packages/odyssey-react-mui/src/Button.tsx

```diff
@@ -154,6 +155,10 @@ const Button = ({
 }: ButtonProps) => {
   const muiProps = useMuiProps();
   const localButtonRef = useRef<HTMLButtonElement>(null);
+  const buttonContext = useButton();
+  const isFullWidth = buttonContext.isFullWidth
+    ? buttonContext.isFullWidth
+    : isFullWidthProp;
```

Comment:
Should this be wrapped in a `useRef` so we don't have to recalculate this each time?